spark/processing/3.0/py3/hadoop-config/spark-defaults.conf

```diff
@@ -4,3 +4,4 @@ spark.executor.extraClassPath    /usr/lib/hadoop-lzo/lib/*:/usr/lib/hadoop/hadoo
 spark.executor.extraLibraryPath  /usr/lib/hadoop/lib/native:/usr/lib/hadoop-lzo/lib/native
 spark.driver.host=sd_host
 spark.hadoop.mapreduce.fileoutputcommitter.algorithm.version=2
+spark.rpc.askTimeout=300s
```

Comment:
Similar to the config changes: could you add comments or references for this config change? I'm hesitant about config changes in the base layer since they could cause unintended changes with the next release.

Reply:
This one should be safe since it only increase the timeout.  Currently we are relying on our integ tests to guarantee the container not broken. For other unforeseen issue we can monitor the canary tests.